google-postgresql.yml

```diff
@@ -39,7 +39,7 @@ provision:
     required: true
     type: number
     details: Size of storage volume for service instance.
-    defualt: 10
+    default: 10
```

Comment:
I would add this to the release notes as a fix

Reply:
ACK